src/templates/static/legal/tac_sections/tac.jsx

```diff
@@ -36,7 +36,7 @@ const Tac = () => (
         <p>{it.L('Acquisition of financial contracts through this site must be effected in accordance with the terms and conditions presented on the pages of this site. Acquisition of a financial contract is completed when the financial contract has been customised, the premium (or the payout, as the case may be) has been calculated and payment has been verified.')}</p>
 
         <h2>F. {it.L('Client\'s Liability')}</h2>
-        <p>{it.L('Clients agree to be fully and personally liable for the due settlement of every transaction entered into under their account with the Company. This includes any transactions entered into by members of the Client\'s family or entourage who have gained access to the account. Clients are responsible for ensuring that they alone control access to the account, and that no minors are granted access to trading on the website. In any case, Clients remains fully liable for any and all positions traded on their account, and for any credit card transactions entered into the site for their account. Each Client also indemnifies the Company in respect to all costs and losses of any kind, whatsoever as may be incurred by the Company as a result, direct or indirect, of the Client\'s failure to perform or settle such a transaction. Online Gambling debts are enforceable in law in the Isle of Man.')}</p>
+        <p>{it.L('Clients agree to be fully and personally liable for the due settlement of every transaction entered into under their account with the Company. This includes any transactions entered into by members of the Client\'s family or entourage who have gained access to the account. Clients are responsible for ensuring that they alone control access to the account, and that no minors are granted access to trading on the website. In any case, Clients remain fully liable for any and all positions traded on their account, and for any credit card transactions entered into the site for their account. Each Client also indemnifies the Company in respect to all costs and losses of any kind, whatsoever as may be incurred by the Company as a result, direct or indirect, of the Client\'s failure to perform or settle such a transaction. Online Gambling debts are enforceable in law in the Isle of Man.')}</p>
```

Comment:
the sentence "Online Gambling debts are enforceable in law in the Isle of Man." is coming out of no-where. Please discuss with Joanna. Either it should be removed, or if it needs to stay, then it needs context (i.e. "Clients who have accounts with BIOM.....")